ruby/command-t/scanner/file_scanner/find_file_scanner.rb

```diff
@@ -36,8 +36,9 @@ def paths!
             counter = 1
             next_progress = progress_reporter.update(counter)
             stdout.each_line do |line|
-              next if path_excluded?(line.chomp!)
-              paths << line[@prefix_len..-1]
+              line = line[@prefix_len..-2]
```

Comment:
Yup, I can add a comment (it's probably worth commenting why both endpoints are what they are). Or but it back if you prefer.